pkgs/applications/networking/p2p/transgui/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchsvn, pkgconfig, makeDesktopItem, unzip, fpc, lazarus,
+libX11, glib, gtk, gdk_pixbuf, pango, atk, cairo }:
+
+stdenv.mkDerivation rec {
+  name = "transgui-trunk-r${version}";
+  version = "986";
```

Comment:
This name won't work well with the version parser, it'll parse as an unversioned package named "transgui-trunk-r986".